contracts/connectors/loantoken/LoanTokenLogicStandard.sol

```diff
@@ -7,16 +7,15 @@ pragma solidity 0.5.17;
 pragma experimental ABIEncoderV2;
 
 import "./AdvancedToken.sol";
+import "./LoanTokenSettingsLowerAdmin.sol";
 import "./interfaces/ProtocolLike.sol";
 import "./interfaces/FeedsLike.sol";
 
 
-contract LoanTokenLogicStandard is AdvancedToken {
-    using SafeMath for uint256;
+contract LoanTokenLogicStandard is AdvancedToken, LoanTokenSettingsLowerAdmin {
```

Comment:
LoanTokenSettingsLowerAdmin is inheriting from AdvancedTokenStorage, AdvancedToken is also inheriting from AdvancedTokenStorage.... I think it's better to let LoanTokenSettingsLowerAdmin inherit from AdvancedToken instead.

Reply:
Please, also remove the function called 'updateSettings'. It's not needed anymore.